Package.swift

```diff
@@ -21,6 +21,12 @@ let package = Package(
             .product(name: "NIOHTTP1", package: "swift-nio"),
             .product(name: "NIOSSL", package: "swift-nio-ssl"),
             .product(name: "NIOWebSocket", package: "swift-nio"),
+            .product(name: "_NIOConcurrency", package: "swift-nio"),
+        ], swiftSettings: [
```

Comment:
These should only be applied to the top level Package.swift (i.e. the one with the executable in it)